blueprints.yaml

```diff
@@ -23,66 +23,81 @@ form:
         0: Disabled
       validate:
         type: bool
-    content_message:
-      type: text
-      label: Message
-      help: Message on the banner
-      placeholder: "This website uses cookies to ensure you get the best experience on our website."
+    assets:
+      type: section
+      title: Assets
+      underline: true
+    built_in_css:
+      type: toggle
+      label: Use built in CSS
+      highlight: 1
+      default: 1
+      options:
+        1: Enabled
+        0: Disabled
       validate:
-        type: text
-    content_dismiss:
-      type: text
-      label: Dismiss button text
-      placeholder: "Got it!"
+        type: bool
+    built_in_js:
+      type: toggle
+      label: Use built in JS
+      highlight: 1
+      default: 1
+      options:
+        1: Enabled
+        0: Disabled
       validate:
-        type: text
-    content_link:
-      type: text
-      label: Policy link text
-      placeholder: "Learn more"
+        type: bool
+    load_js_options_bottom:
+      type: toggle
+      label: Load js options in bottom
+      highlight: 1
+      default: 1
+      options:
+        1: Enabled
+        0: Disabled
       validate:
-        type: text
-    content_href:
+        type: bool
+    custom_document_event:
```

Comment:
I am interested why did you need this custom event? Did you had any trouble with the theme integration?